erpnext/stock/doctype/pick_list/pick_list.json

```diff
@@ -108,10 +110,27 @@
    "options": "STO-PICK-.YYYY.-",
    "reqd": 1,
    "set_only_once": 1
+  },
+  {
+   "depends_on": "eval:doc.purpose==='Delivery against Sales Order'",
+   "fieldname": "customer_name",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "label": "Customer Name",
+   "options": "Customer",
+   "read_only": 1
+  },
```

Comment:
The customer link field already exists, did you mean to add a data field to show in the list view instead?

Reply:
yes i wanted to show customer_name in list view

Reply:
Change the fieldtype to `Data` then and fetch it from the linked customer, we don't need 2 link fields for the same data.